com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/database/JDBCConnectEditView.java

```diff
@@ -114,8 +114,25 @@ private void addDriver() {
 		final String firstFile = fileDialog.open();
 		if (StringUtils.isBlank(firstFile)) {
 			return;
-		}
+		}		
 		if (!JDBCDriverManager.getInstance().addDriver(firstFile, false)) {
+			// --> added by chris
```

Comment:
This comment seems unnecessary. This is because Git knows that this code has been modified from you.